dali/kernels/signal/resampling.h

```diff
@@ -105,6 +147,49 @@ struct Resampler {
     windowed_sinc(window, lookup_size, lobes);
   }
 
+#if defined(__ARM_NEON)
+  inline float filter_vec(int &i_ref, float in_pos, int i1, const float *in) const {
+    const float32x4_t _0123 = vsetq_f32(0, 1, 2, 3);
+    float32x4_t f4 = vdupq_n_f32(0);
+
+    int i = i_ref;
+    float32x4_t x4 = vaddq_f32(vdupq_n_f32(i - in_pos), _0123);
+
+    for (; i + 3 <= i1; i += 4) {
```

Comment:
Shouldn't you change that as well like in L174?

Reply:
True.